packages/async-subscription/src/stateful.ts

```diff
@@ -14,46 +9,26 @@ export function makeStatefulSubscribable<T>(
   // for unsubscribe callbacks received from subscription.subscribe().
   retain(subscription);
 
-  let current = subscription.initial;
-  let listening = true;
-
-  const subscribers = new Set<Subscriber<T>>();
-
-  const subscriptionResult = Promise.resolve<RemoteSubscribeResult<T>>(
-    subscription.subscribe(listener),
-  ).then((result) => {
-    listener(result[1]);
```

Comment:
Just curious here. Does this `listener` ever reach any subscriptions? That is, does the client have time enough to subscribe before this promise resolves?